dash_core_components_base/__init__.py

```diff
@@ -76,18 +76,18 @@
 _js_dist.extend(
     [
         {
-            "relative_package_path": "{}.min.js".format(__name__),
+            "relative_package_path": "{}.js".format(__name__),
```

Comment:
There hasn't been a `dev` vs. `prod` build of this package since we've moved `prop-types` from the component packages to the dash-renderer.